web/src/assets/styles/blocks.scss

```diff
@@ -510,31 +510,6 @@ table.proposal-result {
     );
   }
   /** End of temporary hack */
-
-  @media (width > 768px) {
-    th.details-column {
-      padding-inline-start: calc(60px + var(--spacer-smaller) * 2);
-    }
-
-    td.details-column {
```

Comment:
Do you mind dropping only the reference to these CSS classes in ProposalResultSection.jsx when defining the const `columns`? Thanks in advance.